pr_agent/git_providers/git_provider.py

```diff
@@ -158,10 +161,12 @@ def get_main_pr_language(languages, files) -> str:
                 most_common_extension == 'scala' and top_language == 'scala' or \
                 most_common_extension == 'kt' and top_language == 'kotlin' or \
                 most_common_extension == 'pl' and top_language == 'perl' or \
-                most_common_extension == 'swift' and top_language == 'swift':
+                most_common_extension == 'swift' and top_language == 'swift' or \
+                most_common_extension == top_language:
```

Comment:
i am trying to think if this change is good or not.

@szecsip Have you encountered a case where you needed it ?